CHANGELOG.md

```diff
@@ -31,6 +32,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### 📚 Documentation Updates
 
 - Linked the contributions list (README.md) in the pull request template
+- Updated documentation to reflect the new output and format parameters in the `AccumulationTable` class.
```

Comment:
You don't need to add another entry here. This section of the changelog is used for documentation-only changes, as it's just assumed that we will update the documentation for any other types of changes.